test/common.py

```diff
@@ -2343,6 +2387,26 @@ def browser_open(cls, url):
     if not EMTEST_BROWSER:
       logger.info('No EMTEST_BROWSER set. Defaulting to `google-chrome`')
       EMTEST_BROWSER = 'google-chrome'
+
+    # If only the the browser is specified, use the default arguments used in circleci.
+    browser_args = shlex.split(EMTEST_BROWSER)
+    if len(browser_args) == 1:
+      logger.info('No EMTEST_BROWSER flags set. Defaulting to CI configuration.')
```

Comment:
What should a user pass here to not default to CI configuration, if they don't have any flags to pass?